int64 label
int64 idx
float64 xmin
float64 ymin
float64 xmax
float64 ymax
